README.md

```diff
@@ -165,6 +165,11 @@ echo "Hello SwiftNIO" | nc localhost 9999
 
 If all goes well, you'll see the message echoed back to you.
 
+To generate an xcodeproj to work on SwiftNIO from Xcode 
+```bash
+swift package generate-xcodeproj
+```
+
```

Comment:
It would be good to add another line about what the command does and how to open the project. Something like:

> This generates an Xcode project using SwiftPM. You can open the project with
> ```bash
> open swift-nio.xcodeproj
> ```